lib/plug/crypto/message_verifier.ex

```diff
@@ -8,11 +8,13 @@ defmodule Plug.Crypto.MessageVerifier do
   tampered with.
   """
 
+  @delimiter "++"
```

Comment:
Maybe we can use another symbol as delimiter that not present in regular Base64 alphabet?